cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2309,6 +2368,10 @@ int getPodIndexFromPvcName(String pvcName)  {
             return Integer.parseInt(pvcName.substring(pvcName.lastIndexOf("-") + 1));
         }
 
+        int getPodIndexFromPodName(String podName)  {
+            return Integer.parseInt(podName.substring(podName.lastIndexOf("-") + 1));
+        }
```

Comment:
Isn't there a helper method for this in `KafkaCluster` or `KafkaResources`?

Reply:
No as far as I know.

Reply:
This seems to be used on at least two other placeses. So IMHO you should either factor it into one of the Util classes or open a separate issue so that someone does it later.